test/specs/carousel.test.js

```diff
@@ -474,6 +474,44 @@ describe('<Carousel />', () => {
       expect(beforeSlideSpy).toBeCalledWith(0, 1);
       expect(afterSlideSpy).toBeCalledWith(1);
     });
+
+    describe('disableKeyboardControls', () => {
+      it('should move to next slide when pressing right arrow key', () => {
+        const map = {};
+        document.addEventListener = jest.fn((event, cb) => {
+          map[event] = cb;
+        });
+
+        const wrapper = mount(
+          <Carousel>
+            <p>Slide1</p>
+            <p>Slide2</p>
+            <p>Slide3</p>
+          </Carousel>
+        );
+        expect(wrapper).toHaveState({ currentSlide: 0 });
+        map.keydown({ keyCode: 39 });
+        expect(wrapper).toHaveState({ currentSlide: 1 });
+      });
+
+      it('should not move to next slide when pressing right arrow key', () => {
```

Comment:
this is just a nit picky thing - but it might be nice to include the name of prop that is changing the behavior in the assertion so that the reader knows why it should not move to the next slide in this case. something like `should not move to next slide when pressing right arrow key when disableKeyboardControls is set to true`